newlib/libc/sys/hermit/unistd.c

```diff
@@ -13,6 +13,10 @@ void _exit(int status) {
 	sys_exit(status);
 }
 
+unsigned alarm(unsigned __secs) {
```

Comment:
Let's stick to POSIX:

```suggestion
unsigned alarm(unsigned seconds) {
```

Reply:
I implemented it how it was in the respective headerfile. Should this then also be changed in `unistd.h`?

Reply:
> Should this then also be changed in `unistd.h`?

No, since that file is not Hermit-specific. But for Hermit-specific files, we usually stick with POSIX wordings.

Reply:
Alright, I got it, thanks :)